sql/moz-fx-data-marketing-prod/ga_derived/blogs_sessions_v2/query.sql

```diff
@@ -0,0 +1,205 @@
+--Get all page views with the page location, and a flag for whether it was an entrance or not to the session
+WITH all_page_views AS (
+  SELECT
+    PARSE_DATE('%Y%m%d', event_date) AS date,
```

Comment:
Perhaps we should wrap `date` inside ` `` ` as it's a keyword?

Reply:
Good suggestion, will do

Reply:
Just pushed a fix for this